src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Circle extends ColorSupplier implements Figure {
+    private double radius = Math.random() * 10;
+
+    @Override
+    public String informationFigure() {
+        System.out.printf("Figure: circle, area: %.1f", getArea());
+        System.out.printf(" sq.units, radius: %.1f", radius);
+        System.out.printf(" units, diagonal: %.1f", radius * 2);
+        System.out.println(" units, color: " + getRandomColor());
```

Comment:
```suggestion
        System.out.printf("Figure: circle, area: %.1f sq.units, radius: %.1f units, diagonal: %.1f units, color: %s%n", getArea(), radius, radius * 2, getRandomColor());
```
same with other `printf`